### PR TITLE
Add python API for some math operators.

### PR DESCRIPTION
**Pull request by wanghaoshuang**

### CODE REVIEW

**Comment by reyoung** [comment]
Why not to use `a < b` instead?

```python
a = fluid.layers.data()
b = fluid.layers.data()
cond = a < b
```

**Comment by wanghaoshuang** [comment]
@reyoung This pr is just in order to keep consistency between documents and python API.
Yeah. `a < b` is better than `less_than`.

File: paddle/fluid/API.spec [hunk 1]
```diff
@@ -271,6 +271,10 @@ paddle.fluid.layers.reciprocal ArgSpec(args=['x', 'name'], varargs=None, keyword
 paddle.fluid.layers.square ArgSpec(args=['x', 'name'], varargs=None, keywords=None, defaults=(None,))
 paddle.fluid.layers.softplus ArgSpec(args=['x', 'name'], varargs=None, keywords=None, defaults=(None,))
 paddle.fluid.layers.softsign ArgSpec(args=['x', 'name'], varargs=None, keywords=None, defaults=(None,))
+paddle.fluid.layers.not_equal ArgSpec(args=['x', 'name'], varargs=None, keywords=None, defaults=(None,))
```
Comment (panyx0718) [inline paddle/fluid/API.spec:274]: Why is there only `x`? I thought is should be `x != y`

Comment (wanghaoshuang) [reply paddle/fluid/API.spec:274]: It seems that [generate_layer_fn](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/layer_function_generator.py#L122) is invalid for the operator with multiple inputs. I will check it.

**Pull request closed without merging**